### PR TITLE
Add helper methods to handle ICMPv6TypeCode

### PR DESCRIPTION
**Pull request by randstr**
Fixes issue #84

### CODE REVIEW

File: layers/icmp6.go [hunk 1]
```diff
@@ -91,6 +99,14 @@ func (a ICMPv6TypeCode) String() string {
 	return fmt.Sprintf("%s(%s)", typeStr, codeStr)
 }
 
+func (a ICMPv6TypeCode) SerializeTo(bytes []byte) {
+	binary.BigEndian.PutUint16(bytes, uint16(a))
+}
+
+func CreateICMPv6TypeCode(typ uint8, code uint8) ICMPv6TypeCode {
```
Comment (gconnell) [inline layers/icmp6.go]: please add brief godoc comments on the new exported methods.

**Comment by gconnell** [comment]
Great stuff!  This should be far more readable :)  Just a minor comment about godoc, and we should be good to go.

**Comment by randstr** [comment]
Hold up please, I'd like to change the go syntax representation of ICMPv6TypeCode to something not using curly braces.

**Comment by randstr** [comment]
Done...

**Comment by randstr** [comment]
To be perfectly correct I think we should also be using the reflect package to print the string representation of the type name... <type string using reflect>(%d, %d)

**Comment by randstr** [comment]
I'll leave the GoString() method to a separate pull request.

Please review/merge this one.

**Pull request merged**